src/boot/z_std_dma.c

```diff
@@ -351,16 +352,14 @@ s32 DmaMgr_SendRequestImpl(DmaRequest* req, u32 ram, u32 vrom, u32 size, u32 unk
     req->notifyQueue = queue;
     req->notifyMsg = msg;
 
-    if (1) {
-        if ((sDmaMgrQueueFullLogged == 0) && (sDmaMgrMsgQueue.validCount >= sDmaMgrMsgQueue.msgCount)) {
-            sDmaMgrQueueFullLogged++;
-            osSyncPrintf("%c", BEL);
-            osSyncPrintf(VT_FGCOL(RED));
-            osSyncPrintf("dmaEntryMsgQが一杯です。キューサイズの再検討をおすすめします。");
-            LOG_NUM("(sizeof(dmaEntryMsgBufs) / sizeof(dmaEntryMsgBufs[0]))", ARRAY_COUNT(sDmaMgrMsgs),
-                    "../z_std_dma.c", 952);
-            osSyncPrintf(VT_RST);
-        }
+    if (1 && (sDmaMgrQueueFullLogged == 0) && MQ_IS_FULL(&sDmaMgrMsgQueue)) {
```

Comment:
I think I'll stick with `1 &&` since it's also that way higher up in this function and I think that one has to be that way since it's in a nested condition? If so then assuming they come from the same thing it would make sense for them both to be this way.